examples/complex_wallet_examples/create_faucet_server/internal/methods/address.go

```diff
@@ -0,0 +1,85 @@
+package methods
+
+import (
+	"context"
+	"encoding/base64"
+
+	ec "github.com/bsv-blockchain/go-sdk/primitives/ec"
+	"github.com/bsv-blockchain/go-sdk/script"
+	sdk "github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-wallet-toolbox-faucet-server/internal/constants"
+	"github.com/bsv-blockchain/go-wallet-toolbox-faucet-server/internal/utils"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/brc29"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/defs"
+	"github.com/go-softwarelab/common/pkg/to"
+)
+
+const (
+	balanceListLimit = uint32(100)
+)
+
+// DeriveAddress returns a faucet BRC-29 address
+func DeriveAddress(faucetKeyHex string, network defs.BSVNetwork) (string, error) {
+	priv, err := ec.PrivateKeyFromHex(faucetKeyHex)
+	if err != nil {
+		return "", err
+	}
+
+	identityKey := priv.PubKey()
+
+	derivationPrefixBytes, err := utils.BytesFromBase64(constants.DefaultBase64Prefix)
+	if err != nil {
+		return "", err
+	}
+	derivationSuffixBytes, err := utils.BytesFromBase64(constants.DefaultBase64Suffix)
+	if err != nil {
+		return "", err
+	}
+
+	keyID := brc29.KeyID{
+		DerivationPrefix: base64.StdEncoding.EncodeToString(derivationPrefixBytes),
+		DerivationSuffix: base64.StdEncoding.EncodeToString(derivationSuffixBytes),
+	}
```

Comment:
this is confusing. As prefix and suffix aren't configurable - becuase they are in constants.
I would simply assign those constants to KeyID fields.